GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="7e053cd8-2f6f-46f0-b7bb-b27448358047"
+    Publisher="CN=collabera"
```

Comment:
@nishitha-burman @peiche-jessica - change to CN=microsoft?

todo: see what's in other project dirs .appmanifest files

Reply:
@jasonstephen15 - Delete this "Package.appxmanifest" file?  Most projects in this repo seem to omit it (checking).

Reply:
I changed to "Microsoft" in the two spots in this file, built & ran ok.  So I might Resolve thread.